src/app/content/highlights/components/DiscardModal.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import styled from 'styled-components/macro';
+import Button from '../../../components/Button';
+import theme from '../../../theme';
+import DiscardCard, { Footer } from './DiscardCard';
+
+// tslint:disable-next-line:variable-name
+const Mask = styled.div`
```

Comment:
did you get any design guidance from anyone for this stuff?

Reply:
I only assumed that since the text format matches the error modal, it should look the same way, my bad.
https://openstax.slack.com/archives/C8V1XET1R/p1581460059190200